FilecoinFoundationWeb/filecoin-foundation
# UXIT-1422 - Enforce Type-only imports and exports

## 📝 Description

Added `verbatimModuleSyntax: true` to tsconfig to inforce type-only imports and exports

- **Type:** Refactor

## 🛠️ Key Changes

- new `.tsconfig `compiler option 
- refactor all imports to include type if applicable

## 🧪 How to Test

- **Steps to Test:**
  1. check if you get an error if you delete a `type` keyword from example` import type { Event } from '@/types/eventType'`


## 📸 Screenshots

https://github.com/user-attachments/assets/260d4c19-8ef0-4319-8124-17d7f73e6995

## Resources

https://www.typescriptlang.org/tsconfig/#verbatimModuleSyntax
